src/pal/src/thread/thread.cpp

```diff
@@ -2633,6 +2633,8 @@ CPalThread::GetStackBase()
     status = pthread_attr_get_np(thread, &attr);
 #elif HAVE_PTHREAD_GETATTR_NP
     status = pthread_getattr_np(thread, &attr);
+#elif __ANDROID__
```

Comment:
That was an easy one:
```
check_library_exists(pthread pthread_getattr_np "" HAVE_PTHREAD_GETATTR_NP)
```

`pthread` doesn't exist as a separate library on Android but as part of bionic; I'll update the CMakeFile accordingly.